helm-configs/nova/nova-helm-overrides.yaml

```diff
@@ -863,7 +863,16 @@ conf:
     filter:audit:
       paste.filter_factory: keystonemiddleware.audit:filter_factory
       audit_map_file: /etc/nova/api_audit_map.conf
-  policy: {}
+  policy:
+    "os_compute_api:os-lock-server:unlock:unlock_override": "!"
```

Comment:
Do we need this one to be `role:admin_or_owner`?

Reply:
So this will Unlock a server, regardless who locked the server. The default for this is context_is_admin that would allow nova admin to unlock, we can leave it default to let admin unlock if that is better ?